pkgs/package-overrides.nix

```diff
@@ -120,6 +120,15 @@ in
                 "ext/dom/tests/bug80268.phpt"
               ];
             })
+          ] ++ lib.optionals (lib.versionOlder prev.php.version "7.3" && lib.versionAtLeast prev.php.version "7.1") [
+            ./patches/php71-libxml212-tests.patch
```

Comment:
Could you please add comments with source for the patches?

Reply:
I actually made them myself, adapted from their counterpart for PHP 8.1.